src/index.css

```diff
@@ -51,6 +51,18 @@
       border-top: rgba(0, 0, 0, 0.05) 1px solid;
       box-shadow: 0 5px 8px rgba(0, 0, 0, 0.15);
 
+      &.simple-select,
+      &.radio-select {
+        .checked .node-label,
+        .partial .node-label {
+          font-weight: 600;
+        }
+      }
+
+      &.simple-select .checkbox-item {
+        display: none;
```

Comment:
It’s more easily styled this way. Css is cascading so having the style deep down on a label is not something easily styled

Reply:
Unless it's fairly difficult to style the existing way, I would lean towards keeping the existing implementation. I'm not sure if anyone is relying on it so it's hard to tell. But if we're changing it, this needs to be called out and would constitute a breaking change. And, if we're gonna do it, I would do it in a separate PR with `BREAKING CHANGE: <rest of the title>` so that semantic-release can bump the versions accordingly. Again, if you think the change is worth going through all of that, then I'm up for it.

Reply:
The only, simple way, now is to supply your own className to the component. Unless all your components use simple-select. See: https://stackoverflow.com/questions/2326499/apply-css-styles-to-an-element-depending-on-its-child-elements

I've restored the old, on checkbox-level, styling to avoid any breaking change but also supplied it on dropdownlevel for easier styling in the future. This also adds a lot of more bytes to the ouput since every checkbox needs to have the classname instead of just the dropdown.

Reply:
Can you show me some code examples of before/after using your approach? Just want to be sure I'm looking at the same thing that you are. 

> The checkbox-level targeting also adds more to the ouput since every checkbox needs to have the classname instead of just the dropdown.

GZip can take care of that (which is how I convince myself that the excessive div mania induced by React is OK).

However, I'm totally up for improving the status quo. I do think this merits its own discussion though, don't you? Breaking change or not, if you believe that we can simplify the existing CSS then let's go for it (not here as this is a big change already).

Reply:
You need to check the latest commit. This only shows the outdated css still.

Anyway if you want to style, e.g. simpleSelects labels differently than the regular tree and use both you won’t be able to. That’s unless you provide a class by yourself.
```jsx
<ReactDrop... className=”customsingle” />
```

```css
.react-dropdown-tree-select.customsingle {
    .dropdown-content label {
        color: blue;
    }
}
```

After adding the className on dropdown, for single and radio, you can now instead do (without any passed className):
```css
.react-dropdown-tree-select .simple-select {
    .dropdown-content {
        label {
            color: blue;
        }
        .checkbox-item {
            display: none;
        }
    }
}
```

There’s no possible way to style anything differently with only css without this additional className on the dropdown since you can’t write a selector targeting a parent (label) in css which depends of a child (.checkbox-item.simple-select). The only way is wrapping the component or passing a className which you can target.

With the addition it would make the additional checkbox className obsolete.